Source/Android/app/build.gradle

```diff
@@ -109,8 +109,8 @@ dependencies {
     // For REST calls
     implementation 'com.android.volley:volley:1.2.1'
 
-    // For loading huge screenshots from the disk.
-    implementation 'com.squareup.picasso:picasso:2.71828'
+    // For loading screenshots from the disk.
```

Comment:
This isn't actually what we use it for. (Originally when Picasso was added, games were associated with a screenshot automatically taken when playing the game, but after not too long we switched to using cover art from GameTDB.com.)

Reply:
Ok, I'll change the comment to reflect that